scripts/build-controller-release.sh

```diff
@@ -55,6 +62,11 @@ Environment variables:
   SERVICE_CONTROLLER_SOURCE_PATH:       Path to the service controller source code
                                         repository.
                                         Default: ../{SERVICE}-controller
+  ACK_GENERATE_OLM:                     Enable Operator Lifecycle Manager generators.
+                                        Default: false
+  ACK_GENERATE_OLMCONFIG_PATH:          Path to the service OLM configuration file. Ignored
+                                        if ACK_GENERATE_OLM is not true.
+                                        Default: {SERVICE_CONTROLLER_SOURCE_PATH}/{SERVICE}-olmconfig.yaml
```

Comment:
Would it be reasonable to put all configuration and output for olm in a `olm/` folder inside the service controller source path?

Reply:
Sure, I don't see why not. And `ack-generate` accepts a new location for the file. https://github.com/aws-controllers-k8s/code-generator/blob/main/cmd/ack-generate/command/olm.go#L52-L54. The only reason I placed it at the base of the directory is because that's where the generator configs seemed to live.

Should I go ahead and adjust this default to `{SERVICE_CONTROLLER_SOURCE_PATH}/olm/{SERVICE}-olmconfig.yaml`?

Reply:
I personally dislike polluting the root directory, especially with non-core ACK configuration. But I'll leave that decision up to @jaypipes , since I know he has a master plan for where everything should live.

Reply:
> Should I go ahead and adjust this default to `{SERVICE_CONTROLLER_SOURCE_PATH}/olm/{SERVICE}-olmconfig.yaml`?

Almost :) No need for the `{SERVICE}` part, since the `{SERVICE_CONTROLLER_SOURCE_PATH}` contains the service's name. So, this would work:

`{SERVICE_CONTROLLER_SOURCE_PATH}/olm/olmconfig.yaml`